builds/service/service-deployment.yaml

```
@@ -0,0 +1,80 @@
+# Run a simple smoke test against a dogfood deployment of the IoT Hub service
+
+pr: none
+trigger: none
+
```

Suggest to add this to avoid download source code from repo since this build is not required.
```suggestion
variables:
  Build.SyncSources: false
```

---

Good catch...I forgot to migrate that setting. But wouldn't `- checkout: none` by more appropriate? Docs say that `Build.SyncSources` is specific to TFS2017, not Azure Pipelines.

---

Good to know.  I will update stress and long haul yaml as well. 👍